### ISSUE TITLE
The attention mask is not set and cannot be inferred from input because pad token is same as eos token.As a consequence, you may observe unexpected behavior. 

### ISSUE THREAD

**Issue by AlexandreArantes32** [BAAI-DCAI/M3D#15]
I get the following error :
"The attention mask is not set and cannot be inferred from input because pad token is same as eos token.As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
You are not running the flash-attention implementation, expect numerical differences."

I already tried "model.generation_config.pad_token_id = tokenizer.pad_token_id" or "pad_token_id=generator.tokenizer.eos_token_id".
I'm just running the QuickStart script

**Comment by baifanxxx** [comment]
Hi,

Thank you for your attention. Please try a lower transformers version.

**Comment by EDGSCOUT-li** [comment]
which version？

**Comment by sydat2701** [comment]
Hi, have you guys shown this problem? I'm facing the same and tried to install several transformers versions but nothing work!
